### ISSUE TITLE
Autofocus not working with ->sidebarCollapsibleOnDesktop()

### ISSUE THREAD

**Issue by njxqlus** [laravel-filament/filament#8296]
### Package

filament/filament

### Package Version

v3.0.39

### Laravel Version

v10.22.0

### Livewire Version

v3.0.1

### PHP Version

8.1

### Problem description

`->autofocus()` doesn't make any effect when `->sidebarCollapsibleOnDesktop()` option enabled in `AdminPanelProvider`

### Expected behavior

Autofocus on field on page/form open

### Steps to reproduce

```php
class AdminPanelProvider extends PanelProvider
{
    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
            ->sidebarCollapsibleOnDesktop()
           ...
```

```php
public static function form(Form $form): Form
{
    return $form
        ->schema([
            Forms\Components\TextInput::make('foo')
                ->autofocus(),
            Forms\Components\TextInput::make('bar')
        ]);
}
```

Please use the **autofocus-bug** branch in the reproduction repository.

### Reproduction repository

https://github.com/njxqlus/filamentphp-bug-report/tree/autofocus-bug

**Comment by njxqlus** [comment]
In continuation https://discord.com/channels/883083792112300104/1142039778196455424

**Comment by danharrin** [comment]
Should be fixed by #8792
